docs/installation/presto-admin-installation.rst

```diff
@@ -38,25 +38,24 @@ of the nodes in the cluster.
 .. NOTE::
      For Amazon EMR, use the online installer instead of the offline installer.
    
-2. Extract and sudo run the installation script from within the ``prestoadmin`` directory.
+2. Extract and run the installation script from within the ``prestoadmin`` directory.
 ::
 
  $ tar xvf prestoadmin-1.5-offline.tar.bz2
  $ cd prestoadmin
- $ sudo ./install-prestoadmin.sh
+ $ ./install-prestoadmin.sh
 
 The installation script will create a ``presto-admin-install`` directory and an
-executable ``presto-admin`` script. Make sure to run the installation script
-with sudo when the presto-admin user is non-root.
-
-By default, the ``presto-admin`` config and log directory locations are
-configured to be at ``~/.prestoadmin`` and ``~/.prestoadmin/log``, respectively.  This
-can be changed by modifying the environment variables, PRESTO_ADMIN_CONFIG_DIR and
-PRESTO_ADMIN_LOG_DIR.
+executable ``presto-admin`` script. By default, the ``presto-admin`` config and log
+directory locations are configured to be ``~/.prestoadmin`` and ``~/.prestoadmin/log``,
+respectively.  This can be changed by modifying the environment variables,
+PRESTO_ADMIN_CONFIG_DIR and PRESTO_ADMIN_LOG_DIR. The installation script will also create
```

Comment:
I would say  "The installation script will create the appropriate configuration and log directories for presto-admin".  I'd leave out the second part of the sentence about using sudo.  I think it's clear enough that if you're doing something that requires sudo privileges, you should use sudo.  You'll get an error message if you mess up.  and i think the docs should be optimized for being clear for the default configurations, for which you definitely don't want to use sudo or those directories will be created in the wrong place.